mkdocs.yml

```diff
@@ -16,28 +17,50 @@ theme:
   name: material
   language: en
   palette:
-    primary: blue
-    accent: blue
+    # Light mode
```

Comment:
According to your feedback, I see that the missing parts here are:
* auto-invalidate applied settings when the OS-wide scheme was changed
* or even to reflect the OS-wide scheme switching

Let's ping @squidfunk because he might be interested in `mkdocs-material` feedback/feature request.

Reply:
On the first visit, system preferences are read (when used together with media queries as described in the docs) and written to local storage. On the next visit, this setting is used. There's no dedicated "system preferences" mode and no plans to support it.